internal/adapters/repositories/user_repo.go

```diff
@@ -38,6 +38,15 @@ func (uri *UserRepositoryImpl) FindUserByID(id pgtype.UUID) (*response.FindUserR
 
 	return response, nil
 }
+func (uri *UserRepositoryImpl) FindUserByPhone(phone string) (*response.FindUserResponse, error){
+	response, err := uri.Queries.FindUserByPhone(phone)
+	
+	if err != nil{
+		return nil, err
+	}
+
+	return response, nil
+}
```

Comment:
if this function do nothing. You can return it like `return uri.Queries.FindUserByPhone(phone)`cuz in usecase layer, you do this err check too (no need to do the same check 2 times or more)